src/averageAges.js

```diff
@@ -20,6 +20,25 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const isCenturySpecified = arguments.length > 1;
+
+  const callback = (person) => {
+    switch (isCenturySpecified) {
+      case true:
+        return Math.ceil(person.died / 100) === century
+          ? [person.name, person.died - person.born]
+          : null;
+
+      case false:
+        return [person.name, person.died - person.born];
```

Comment:
do you need person's name in this array? just return age

```suggestion
        return person.died - person.born;
```